modules/openideal_statistics/src/Plugin/Block/OpenidealStatisticsIdeaStatisticsBlock.php

```diff
@@ -79,6 +85,27 @@ public function build($challenge = NULL) {
       ],
     ];
 
+    // Do not show if idea is not in below then expert review workflow phase.
+    if ($this->configuration['show_five_stars'] && $node->bundle() == 'idea' &&
```

Comment:
Can we add a method `isExpertReview` to IdeaHelper class?

Reply:
Yes, but it's a unique usage of this condition. Maybe if we will have more than one we can implement this method?

Reply:
Fair enough.